### PR TITLE
fix inline style

### PR DESCRIPTION
**Pull request by Inbal-Tish**
Run 'npm run lint' to see rest of the warnings

### CODE REVIEW

File: example/src/screens/calendars.js [hunk 1]
```diff
@@ -355,16 +355,12 @@ const CalendarsScreen = () => {
...
-              borderBottomWidth: 1,
-              borderBottomColor: 'lightgrey'
-            }
+            styles.customCalendar
           ]}
           dayComponent={({date, state}) => {
             return (
               <View>
-                <Text style={{textAlign: 'center', color: state === 'disabled' ? 'gray' : 'purple'}}>{date.day}</Text>
+                <Text style={[styles.customDay, {color: state === 'disabled' ? 'gray' : 'purple'}]}>{date.day}</Text>
```
Comment (ethanshar) [inline example/src/screens/calendars.js:363]: you can improve here a little and create `styles.disableDay` & `styles.day` and pass them in the ternary.

File: example/src/screens/calendarsList.js [hunk 2]
```diff
@@ -25,16 +25,10 @@ const CalendarsList = () => {
...
-              width: '100%',
-              justifyContent: 'space-between',
-              marginTop: 10,
-              marginBottom: 10
-            }}
+            style={styles.header}
           >
-            <Text style={{marginLeft: 5, ...textStyle}}>{`${month}`}</Text>
-            <Text style={{marginRight: 5, ...textStyle}}>{year}</Text>
+            <Text style={[styles.month, {...textStyle}]}>{`${month}`}</Text>
```
Comment (ethanshar) [inline example/src/screens/calendarsList.js:30]: Spreading a object inside an empty object (`{...textStyle}`) just creates a new object each time and breaking reference comparison.

File: src/expandableCalendar/index.js [hunk 3]
```diff
@@ -445,13 +445,7 @@ class ExpandableCalendar extends Component {
...
       <Animated.View
         ref={e => (this.weekCalendar = e)}
-        style={{
-          position: 'absolute',
-          left: 0,
-          right: 0,
-          top: HEADER_HEIGHT + (commons.isAndroid ? 8 : 4), // align row on top of calendar's first row
-          opacity: position === POSITIONS.OPEN ? 0 : 1
-        }}
+        style={[this.style.weekContainer, {opacity: position === POSITIONS.OPEN ? 0 : 1}]}
```
Comment (ethanshar) [inline src/expandableCalendar/index.js:448]: Also here, you can create styles for `hidden` (0) and visible (1) and use it in the ternary. 
even better, the default opacity is 1, so instead you can just do `position === POSITIONS.OPEN && styles.hidden`, no need for ternary

File: src/expandableCalendar/style.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
 import {StyleSheet, Platform} from 'react-native';
 import * as defaultStyle from '../style';
-
+import {HEADER_HEIGHT} from './index.js';
```
Comment (ethanshar) [inline src/expandableCalendar/style.js:3]: importing from index can create cyclic require. 
Consider moving the constants to a 3rd file, or make sure the imports are in one direction (maybe declare it in styles and import it from there in the index)

File: example/src/screens/calendars.js (hunk 1 above)
Comment (Inbal-Tish) [reply example/src/screens/calendars.js]: Done

File: example/src/screens/calendarsList.js (hunk 2 above)
Comment (Inbal-Tish) [reply example/src/screens/calendarsList.js]: True. Done

File: src/expandableCalendar/index.js (hunk 3 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/index.js]: Done

File: src/expandableCalendar/style.js (hunk 4 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/style.js]: Nice! Done

**Review by ethanshar — APPROVED** [review]

**Pull request merged**